app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt

```diff
@@ -15,28 +15,55 @@ class SocialLinksViewHolder(itemView: View, private var context: Context) : Recy
 
     fun bind(socialLink: SocialLink) {
         val drawableId = getSocialLinkDrawableId(socialLink.name)
-        if (drawableId != -1) {
-            val imageDrawable: Drawable? = ContextCompat.getDrawable(context, drawableId)
-            imageDrawable?.colorFilter = PorterDuffColorFilter(ContextCompat.getColor(context, R.color.greyMore),
-                PorterDuff.Mode.SRC_IN)
-
-            itemView.imgSocialLink.setImageDrawable(imageDrawable)
-        }
+        val imageDrawable: Drawable? = ContextCompat.getDrawable(context, drawableId)
+        imageDrawable?.colorFilter = PorterDuffColorFilter(ContextCompat.getColor(context, R.color.greyMore),
+            PorterDuff.Mode.SRC_IN)
+        itemView.imgSocialLink.setImageDrawable(imageDrawable)
 
         itemView.setOnClickListener {
             Utils.openUrl(context, socialLink.link)
         }
     }
 
     private fun getSocialLinkDrawableId(name: String): Int {
-        return when (name) {
-            "Github Url" -> R.drawable.ic_github
-            "Twitter Url" -> R.drawable.ic_twitter
-            "Facebook Url" -> R.drawable.ic_facebook
-            "LinkedIn Url" -> R.drawable.ic_linkedin
-            "Youtube Url" -> R.drawable.ic_youtube
-            "Google Url" -> R.drawable.ic_google_plus
-            else -> -1
+        return when (name.toLowerCase()) {
+            "github url",
+            "githuburl",
+            "github" -> R.drawable.ic_github
+            "twitter url",
+            "twitterurl",
+            "twitter" -> R.drawable.ic_twitter
+            "facebook url",
+            "facebookurl",
+            "facebook" -> R.drawable.ic_facebook
+            "linkedin url",
+            "linkedinurl",
+            "linkedin" -> R.drawable.ic_linkedin
+            "youtube url",
+            "youtubeurl",
+            "youtube" -> R.drawable.ic_youtube
+            "googleplus",
+            "googleplus url",
+            "googleplusurl",
+            "google plus",
+            "google plusurl",
+            "google plus url",
+            "google url",
+            "googleurl",
+            "google" -> R.drawable.ic_google_plus
+            "wikipedia",
+            "wikipedia url",
+            "wikipediaurl" -> R.drawable.ic_wikipedia
+            "flickr",
+            "flickr url",
+            "flickrurl" -> R.drawable.ic_flickr
+            "blogger",
+            "blogger url",
+            "bloggerurl",
+            "blog",
+            "blog url",
+            "blogurl" -> R.drawable.ic_blogger
+            else -> R.drawable.ic_link_black
```

Comment:
Use `if (name.toLowerCase().includes("github"))` and so on

Reply:
Thanks for reminding, I forgot about includes function.

Reply:
done